pom.xml

```diff
@@ -244,14 +245,14 @@
         <version>${cassandra.plugin.version}</version>
         <configuration>
           <!--There are issues when running tests inside vmware fusion on a filesystem mounted from the host machine-->
-          <cassandraDir>/tmp/bf-cassandra-${cassandra.plugin.version}/</cassandraDir>
+          <cassandraDir>/tmp/bf-cassandra-${cassandra.plugin.version}-${maven.build.timestamp}/</cassandraDir>
```

Comment:
Nah..... disk space is cheap!!!!

Jk, yes, it does. That's why I move this from pre-integration-test phase to post-integration-test phase.